great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -232,6 +233,12 @@ def get_expectation_suite(
 
         """
         if send_usage_event:
+            if not expectation_suite_name:
```

Comment:
@TrangPham Could the method `get_or_create_expectation_suite()` (or parts of it) in "great_expectations/rule_based_profiler/helpers/util.py" be of help here?  Please feel free to refactor that method if you think that you can use parts of it for the present purpose.  Thank you!

Reply:
I thought about refactoring it, but I'm not sure how to handle this with the usage decorator. The usage stats decorator would be passed the same arguments as the function it decorates. 

Do you have suggestions on how to refactor here?

Reply:
@TrangPham My idea was to take this code
```
    if generate_temp_expectation_suite_name:
        if not component_name:
            component_name = "test"

        expectation_suite_name = f"{TEMPORARY_EXPECTATION_SUITE_NAME_PREFIX}.{component_name}.{TEMPORARY_EXPECTATION_SUITE_NAME_STEM}.{str(uuid.uuid4())[:8]}"
```
from the method `get_or_create_expectation_suite()` in "great_expectations/rule_based_profiler/helpers/util.py" and use that inside the `get_expectation_suite()` here -- so that we have a consistent naming of expectation suites.  Would that work?

Reply:
Yes updated. The component_name would not be none here since we do have a class, so I will leave that part off